CONTRIBUTING.md

```diff
@@ -41,19 +41,21 @@ The `actions/` directory contains composite actions. Refer to the [GitHub docume
 
 4. **Make Changes:** Make your desired changes to the codebase.
 
-5. **Document Changes:** Document your changes in a `README.md` file. Try to follow the examples set forward in other Readmes.
+5. **Write Tests:** Where it makes sense, please add tests. See `actions/techdocs-rewrite-relative-links` as an example.
 
-6. **Commit Changes:** Commit your changes with clear and concise commit messages.
+6. **Document Changes:** Document your changes in a `README.md` file. Try to follow the examples set forward in other Readmes.
 
-7. **Push Changes:** Push your changes to your forked repository.
+7. **Commit Changes:** Commit your changes with clear and concise commit messages.
 
-8. **Submit a Pull Request:** Submit a pull request to the main repository. Ensure that your pull request is detailed and includes information about the changes made.
+8. **Push Changes:** Push your changes to your forked repository.
 
-9. **Check CI:** Check your pull request to verify that any CI processes are passing.
+9. **Submit a Pull Request:** Submit a pull request to the main repository. Ensure that your pull request is detailed and includes information about the changes made.
 
-10. **Review Process:** Your pull request will be reviewed by Grafana's platform-productivity squad. Make any requested changes or address any feedback during this process.
+10. **Check CI:** Check your pull request to verify that any CI processes are passing.
 
-11. **Contributor License Agreement (CLA):** Before we can accept your pull request, you need to sign [Grafana's CLA](https://grafana.com/docs/grafana/latest/developers/cla/). If you haven't, our CLA assistant prompts you to when you create your pull request.
+11. **Review Process:** Your pull request will be reviewed by Grafana's platform-productivity squad. Make any requested changes or address any feedback during this process.
```

Comment:
nit: In this case I think we are talking about Grafana Labs

```suggestion
11. **Review Process:** Your pull request will be reviewed by the Grafana Labs' platform-productivity squad. Make any requested changes or address any feedback during this process.
```

somewhat related: we should generally avoid product possessives (https://github.com/grafana/writers-toolkit/blob/main/vale/Grafana/ProductPossessives.yml) but here we are not talking about a product.